python/lsst/pipe/tasks/postprocess.py

```diff
@@ -850,6 +850,10 @@ def run(self, handle, funcs=None, dataId=None, band=None):
 
         df = self.transform(band, handle, funcs, dataId).df
         self.log.info("Made a table of %d columns and %d rows", len(df.columns), len(df))
+
+        if len(df) == 0:
+            raise UpstreamFailureNoWorkFound("No rows were created in upstream task to transform.")
```

Comment:
The error message is not very clear. Perhaps something like:
`"Input catalog is empty, so there is nothing to transform/standardize"`